src/components/cap-volume.js

```diff
@@ -39,27 +99,35 @@ export default class CapVolume extends LitElement {
 	render() {
 		//todo this will need to be updated to deal with multiple cases on the same page
 		return html`
-			<h1>${this.volume} ${this.reporterData.short_name}</h1>
-			<h2>
-				${this.reporterData.full_name}
-				(${this.reporterData.start_year}-${this.reporterData.end_year}) volume
-				${this.volume}
-			</h2>
-			<ul>
-				${this.casesData.map(
-					(c) =>
-						html`<li>
-							<a
-								href="/caselaw/?reporter=${this.reporter}&volume=${this
-									.volume}&case=${String(c.first_page).padStart(4, "0")}"
-							>
-								${c.name_abbreviation},
-								${c.citations.filter((c) => c.type == "official")[0].cite}
-								(${c.decision_date.substring(0, 4)})
-							</a>
-						</li>`,
-				)}
-			</ul>
+			<cap-caselaw-layout>
+				<div class="volumes__main">
+					<hgroup>
+						<h1 class="volume__heading">
+							${this.volume} ${this.reporterData.short_name}
+						</h1>
+						<p>
+							${this.reporterData.full_name}
+							(${this.reporterData.start_year}-${this.reporterData.end_year})
+							volume ${this.volume}.
+						</p>
```

Comment:
I noticed in `cap-reporter.js` that the subheading for each reporter is styled with a font-weight of 500. For consistency, I think it makes sense to do so here for this paragraph element/subheading as well. (Or, alternatively, we could easily remove it)